frontend/src/components/KudosCard.jsx

```diff
@@ -0,0 +1,38 @@
+import "./KudosCard.css";
+import { useState } from "react";
+
+const DEFAULT_GIF = "https://i.gifer.com/4j.gif";
+
+const KudosCard = ({ id }) => {
+  const handleDelete = async () => {
+    const response = await fetch(`http://localhost:3000/boards/${id}`, {
+      method: "DELETE",
+    });
+    const data = await response.json();
+    console.log(data);
```

Comment:
Don't leave console logs. Even in the intermediate commits. Treat every commit as a stable version of your app without bad or redundant code.